src/Akeneo/Pim/Enrichment/Component/Product/Connector/Reader/Database/ProductReader.php

```
@@ -170,13 +170,26 @@ protected function getConfiguredFilter(string $fieldName)
      */
     protected function getProductsCursor(array $filters, ChannelInterface $channel = null)
     {
-        $options = ['filters' => $filters];
-
-        if (null !== $channel) {
-            $options['default_scope'] = $channel->getCode();
-        }
+        $options = null !== $channel ? ['default_scope' => $channel->getCode()] : [];
 
         $productQueryBuilder = $this->pqbFactory->create($options);
+        foreach ($filters as $filter) {
+            try {
+                $productQueryBuilder->addFilter(
+                    $filter['field'],
+                    $filter['operator'],
+                    $filter['value'],
+                    $filter['context'] ?? []
+                );
+            } catch (ObjectNotFoundException $e) {
+                /**
+                 * PIM-8127: If a filter can not be applied because the object can not be found, it is ignored.
+                 * ie: Family or group does not exists.
+                 */
+                $this->stepExecution->addError($e->getMessage());
```

So the exception is ignored but the job will have an error ?